aspnet-core/xml/ns-Microsoft.AspNetCore.Authentication.Facebook.xml

```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.Authentication.Facebook">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Contains types that enable support for facebook OAuth based authentication.</summary>
```

Comment:
```suggestion
    <summary>Contains types that support facebook OAuth based authentication.</summary>
```

Reply:
If you agree with this suggestion, apply globally. Your version is more accurate, mine is more terse.